Dockerfile

```diff
@@ -47,44 +57,43 @@ RUN curl --retry 7 -Lso /tmp/consul-template.zip "https://releases.hashicorp.com
     && unzip /tmp/consul-template.zip -d /usr/local/bin \
     && rm /tmp/consul-template.zip
 
+# Make the WP uploads directory writeable by the web server
+#RUN chown -R www-data:www-data /var/www/html/content/uploads
+
+# install wp-cli, http://wp-cli.org
+ENV WP_CLI_CONFIG_PATH /var/www/html/wp-cli.yml
+RUN curl -Ls -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
+    && chmod +x wp-cli.phar \
+    && mv wp-cli.phar /usr/local/bin/wp \
+    && wp --info --allow-root
+
 # copy the WordPress skeleton from this repo into the container
 # this includes any themes and/or plugins we've added to the content/themes and content/plugins, etc, directories.
 COPY /var/www/html /var/www/html
 
-# Make the WP uploads directory writeable by the web server
-RUN chown -R www-data:www-data /var/www/html/content/uploads
-
-# install WordPress
-# Releases at https://wordpress.org/download/release-archive/
-ENV WORDPRESS_VERSION 4.4.1
-ENV WORDPRESS_SHA1 89bcc67a33aecb691e879c818d7e2299701f30e7
-# upstream tarballs include ./wordpress/ so this gives us /var/www/html/wordpress
-RUN curl -o wordpress.tar.gz -SL https://wordpress.org/wordpress-${WORDPRESS_VERSION}.tar.gz \
-    && echo "${WORDPRESS_SHA1} *wordpress.tar.gz" | sha1sum -c - \
-    && tar -xzf wordpress.tar.gz -C /var/www/html \
-    && rm wordpress.tar.gz
+
+ENV WORDPRESS_VERSION=${WORDPRESS_VERSION:-4.4.2}
```

Comment:
As you'd pointed out, we can't inject environment vars at build time, so this should be the simpler syntax, perhaps?